models/index.js

```diff
@@ -10,8 +10,10 @@ const db = {};
 
 let sequelize;
 if (config.use_env_variable) {
+  console.log('config', config)
   sequelize = new Sequelize(process.env[config.use_env_variable], config);
 } else {
+  console.log('config no env variable', config)
   sequelize = new Sequelize(config.database, config.username, config.password, config);
 }
 
```

Comment:
Importing the models by looping through their file names feels unnecessarily complicated over just adding a new sequelize.import() call when we add a model, but it does remove the step of having to add it when we scaffold a new model I guess.

Reply:
This is actually auto-generated by sequelize cli as their particular solution to this problem.

Reply:
Oh, in that case nevermind